apps/app/src/components/UnlockStore.tsx

```diff
@@ -48,23 +60,32 @@ const UnlockStore: Component = () => {
             onInput={(e: any) => {
               setPassword(e.target.value);
               setError(false);
-            }}
-          />
+            } } />
```

Comment:
formatting

enabling the `format on save` setting in visual studio code will make your code look much nicer :)